apps/web/src/components/Mdx.tsx

```diff
@@ -0,0 +1,17 @@
+import { getMDXComponent } from "next-contentlayer/hooks";
+
+import { components } from "./MdxComponents";
+
+interface MdxProps {
+  code: string;
+}
+
+export function Mdx({ code }: MdxProps) {
+  const MDXComponent = getMDXComponent(code);
+
+  return (
+    <article className="prose prose-quoteless prose-neutral dark:prose-invert max-w-none">
+      <MDXComponent components={{ ...components }} />
+    </article>

```

Comment:
Here you define `prose` twice. Maybe keep that here and remove it from above.

Reply:
I didn't see that, fixed it!